spec/rubocop/cop/layout/else_alignment_spec.rb

```diff
@@ -535,4 +535,36 @@ def my_func
       RUBY
     end
   end
+
+  context 'ensure/rescue/else in Block Argument' do
+    let(:ruby_version) { 2.5 }
+
+    it 'accepts a correctly aligned else' do
+      expect_no_offenses(<<-RUBY.strip_indent)
+        array_like.each do |n|
+          puts 'do something error prone'
+        rescue SomeException
+          puts 'error handling'
+        rescue
+          puts 'error handling'
+        else
+          puts 'normal handling'
+        end
+      RUBY
+    end
+    it 'registers an offense for misaligned else' do
```

Comment:
I prefer inserting a new line between the examples.
For example:

```ruby
# bad
it 'something' do
end
it 'something2' do
end

# good
it 'something' do
end

it 'something2' do
end
```

Probably rubocop-rspec should detect the code.